src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Navigation.propTypes = {
+  handleClick: PropTypes.func,
+  children: PropTypes.string.isRequired,
+}
+
+export default function Navigation({ handleClick, children }) {
+  return <NavButton onClick={handleClick}>{children}</NavButton>
+}
+
+const NavButton = styled.button`
+  background-color: transparent;
+  border: var(--primary-dark) 2px solid;
+  color: var(--primary-dark);
+  padding: 6px;
+  font-size: 0.8em;
+`
```

Comment:
Could you not just reuse your Button Component here? Styling looks similar

Reply:
To structure my code clearly, I think it's better to separate a regular call to action button from a navigation button.